packages/tooling/fast-tooling/src/message-system/webworker.ts

```diff
@@ -3,7 +3,7 @@
 import { getMessage } from "./message-system.utilities";
 
 onmessage = function (e: MessageEvent): void {
-    if (e.data && e.data.type !== undefined) {
+    if (e.data && e.data?.[0]?.type) {
```

Comment:
I think this is checking `e.data` twice right? If so, this could be reduced.

Reply:
Whoops, you're right